tests/test-miss-handlers.lua

```diff
@@ -40,4 +40,8 @@ local w = wibox()
 w.foo = "bar"
 assert(w.foo == "bar")
 
+-- Test if read-only properties really are read-only
+screen[1].clients = 42
+assert(screen[1].clients ~= 42)
```

Comment:
I guess that means "please squash the fixup commits into the related commits". Ok. :-)